kong/db/dao/init.lua

```diff
@@ -1103,6 +1103,18 @@ function DAO:each(size, options)
 end
 
 
+function DAO:each_for_export(size, options)
+  local iter = self:each(size, options)
+  return function ()
+    local row, err, page = iter()
+    if row and row.ttl and row.ttl ~= null then
+      row.ttl = row.ttl + (row.updated_at or row.created_at or ngx.time())
+    end
+    return row, err, page
+  end
+end
```

Comment:
hi @bungle is `each_for_export` added here ok? I find  `db[name]:each` in `for row, err in db[name]:each` is a DAO object, so I also added one DAO method to reuse `:each` and export absolute ttl.